src/main/java/net/snowflake/ingest/streaming/internal/AbstractRowBuffer.java

```diff
@@ -152,6 +152,8 @@ public int getOrdinal() {
   // Names of non-nullable columns
   private final Set<String> nonNullableFieldNames;
 
+  private final Set<String> collatedColumnsInternalNames;
```

Comment:
Nit, add docstring here. Might be good to clarify what an "internal name" is here.

Reply:
Comment added